kylin/pom.xml

```diff
@@ -34,34 +34,33 @@
     <name>Zeppelin: Kylin interpreter</name>
 
     <dependencies>
+
         <dependency>
             <groupId>${project.groupId}</groupId>
             <artifactId>zeppelin-interpreter</artifactId>
             <version>${project.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.3.6</version>
```

Comment:
@bzz It's safe since 

1. we have this in `root/pom.xml` and all dependencies in `dependencyManagement` block will be inherited.
(ref - https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Management)

```
     <properties>
         <httpcomponents.client.version>4.3.6</httpcomponents.client.version>

    <dependencyManagement>
      <dependency>
        <groupId>org.apache.httpcomponents</groupId>
        <artifactId>httpclient</artifactId>
        <version>${httpcomponents.client.version}</version>
      </dependency>
```

2. You can check using effective pom 

```
$ mvn org.apache.maven.plugins:maven-help-plugin:2.2:effective-pom -pl kylin

// you will see
      <dependency>
        <groupId>org.apache.httpcomponents</groupId>
        <artifactId>httpclient</artifactId>
        <version>4.3.6</version>
      </dependency>
```